.github/workflows/fedara.yml

```diff
@@ -0,0 +1,227 @@
+name: Fedora (RHEL), Python 3.9
```

Comment:
typo in the filename: `fedara.yml` -> `fedora.yml`.

Reply:
good catch, fixed